src/main/java/com/uid2/shared/store/KeysetKeyStoreSnapshot.java

```diff
@@ -7,28 +7,28 @@
 import java.util.*;
 
 public class KeysetKeyStoreSnapshot implements IKeysetKeyStore.IkeysetKeyStoreSnapshot {
-    private final HashMap<Integer, KeysetKey> keyMap;
-    private final HashMap<Integer, List<KeysetKey>> keysetMap;
-    private final List<KeysetKey> activeKeys;
+    private final HashMap<Integer, KeysetKey> keyIdToKeyMap;
+    private final HashMap<Integer, List<KeysetKey>> keysetIdToKeyListMap;
+    private final List<KeysetKey> allKeys;
 
-    public KeysetKeyStoreSnapshot(HashMap<Integer, KeysetKey> keyMap, HashMap<Integer, List<KeysetKey>> keysetMap) {
-        this.keyMap = keyMap;
-        this.keysetMap = keysetMap;
-        this.activeKeys = new ArrayList<>(keyMap.values());
+    public KeysetKeyStoreSnapshot(HashMap<Integer, KeysetKey> keyIdToKeysetKey, HashMap<Integer, List<KeysetKey>> keysetIdToKeysetKeyList) {
+        this.keyIdToKeyMap = keyIdToKeysetKey;
+        this.keysetIdToKeyListMap = keysetIdToKeysetKeyList;
+        this.allKeys = new ArrayList<>(keyIdToKeysetKey.values());
```

Comment:
btw why do we store this, is it too slow to call keyIdToKeysetKey.values() each time ?

Reply:
Just copying how the encryption keys worked